frontend/src/components/toast/toast-input.js

```diff
@@ -152,11 +152,10 @@ class ToastInput extends React.Component {
             type={this.props.type}
             name={this.props.name}
             placeholder={this.props.placeholder}
-            defaultValue={this.props.defaultValue || ""}
+            value={this.props.value || ""}
```

Comment:
defaultValue was used to solve a bug before about not being able to type into the fields. did you specifically test for that when you changed this? and what's the reason for changing this?